ocaml/parsing/jane_syntax_parsing.ml

```diff
@@ -83,11 +83,11 @@ open Parsetree
 
 module Feature : sig
   type t =
-    | Language_extension of Language_extension.t
+    | Language_extension : 'a Language_extension.t -> t
```

Comment:
```suggestion
    | Language_extension : _ Language_extension.t -> t
```

Here and throughout, I've made this suggestion to be more suggestive of a type variable that doesn't corefer.